aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLib.cpp

```diff
@@ -709,12 +709,9 @@ inline static void apply_svd_cusolver_gesvdj(const Tensor& A, const Tensor& U, c
   gesvdjInfo_t gesvdj_params;
   TORCH_CUSOLVER_CHECK(cusolverDnCreateGesvdjInfo(&gesvdj_params));
 
-  // ROCM does not support those functions
-#ifdef CUDART_VERSION
   // Todo: expose the following two parameters to users
   TORCH_CUSOLVER_CHECK(cusolverDnXgesvdjSetTolerance(gesvdj_params, std::numeric_limits<scalar_t>::epsilon()));
```

Comment:
@alugorey It seems there's either a duplication or a typo in the hipify mappings for these functions: https://github.com/alugorey/pytorch/blob/3cad1fb057254136387970ce0e74d354b397ae81/torch/utils/hipify/cuda_to_hip_mappings.py#L8208. Can you please confirm and rectify the mappings as part of this PR?